src/js/component/App.js

```diff
@@ -22,6 +23,7 @@ class App {
     this.ul = document.querySelector(settings.ulSelector)
     this.channelUILists = document.querySelector(settings.channelUIListsSelector)    
     this.channels = {}
+    this.api = new Api("")
```

Comment:
I will suggest to use one instance of api for all appication.
for example
class Api {}

export const api = new Api(BASE_URL + API_KEY) // by writing that string all pieces of your app will use single instance which making calls to single url

Then you can just import it as api and use as api.get('some addditinal url') api.put() api.post()

Reply:
I applied same way last evening - By taking our Main App reference file. But I had been getting some errors. I agree and let me try again.

Reply:
It works and Applied as suggested